### PR TITLE
Harvest org CS Score improvements

### PR DESCRIPTION
**Pull request by n13**
## Spec

From constitution

Orgs need their own scoped tables for community building points, etc

Questionable whether orgs should get planted score but if they do it would also need to be in their own scope

(Org planted score is marked as Version 2.0 feature which means it will be re-evaluated once we shipped 1.0)

Relevant code / comments

```
  auto pitr = planted.find(account.value);
  if (pitr != planted.end()) planted_score = pitr->rank;

  // CS scrore for org needs to be calculated differently
  // Page 71 constitution

  if (type == "organisation"_n) {
    tx_points_tables orgtxpoints(get_self(), "org"_n.value);
    auto titr = orgtxpoints.find(account.value);

.....

  auto ritr = rep.find(account.value);
  if (ritr != rep.end()) reputation_score = ritr->rank;

  // TODO verify this as correct for the constitution pp 71
  // Orgs need to have different scope for rep
  // Orgs need to have different scope for cbp

  // org Planted seeds ranking for orgs may need to be in a different scope (Seeds 2.0 feature)
  // ORG Total Organisation Contribution Point = (CC+EC) * ORM
```

## Deployment
From Ian, below
I changed the scope for organizations to 'org' in the cbs, rep and cs ranks. I added some new tests for that.

### Contracts
accounts
harvest
history
scheduler

### Migration

There are migration methods in `accounts` In `accounts` contract:
- `migorgs`: this action passes the cbs and rep info to the 'org' scope, no deletes any entry in the old scope
and `harvest`, to pass the information for cbs, rep and cs to the new scope:
- `delcbsreporg`: this action erases the cbs and rep entries for users that are an organization

In `harvest` contract:
- `migorgs`: this action passes the cs info to the 'org' scope, no deletes any entry in the old scope
- `delcsorg`: this action erases the cs entries for users that are an organization

### Update Scheduler
I also added new actions for ranking the orgs, and I added them to the scheduler too.

I removed the cbs from organization contract, before it was there but it was repetitive, so the methods organization contract had for adding/removing/ranking cbs points for organizations were removed and instead the methods in harvest and accounts are used.

Also History contract needs to be recompiled to apply the change in utils for the scoped rep

### CODE REVIEW

**Comment by IanMendozaJaimes** [comment]
I changed the scope for organizations to 'org' in the cbs, rep and cs ranks. I added some new tests for that.

There are migration methods in `accounts` and `harvest`, to pass the information for cbs, rep and cs to the new scope:
In `accounts` contract:
- `migorgs`: this action passes the cbs and rep info to the 'org' scope, no deletes any entry in the old scope
- `delcbsreporg`: this action erases the cbs and rep entries for users that are an organization

In `harvest` contract:
- `migorgs`: this action passes the cs info to the 'org' scope, no deletes any entry in the old scope
- `delcsorg`: this action erases the cs entries for users that are an organization

I also added new actions for ranking the orgs, and I added them to the scheduler too.

I removed the cbs from organization contract, before it was there but it was repetitive, so the methods organization contract had for adding/removing/ranking cbs points for organizations were removed and instead the methods in harvest and accounts are used

**Comment by n13** [comment]
@IanMendozaJaimes please review the deployment section I added to the top, so I can deploy easily

File: test/accounts.test.js [hunk 1]
```diff
@@ -1744,3 +1744,109 @@ describe('Punishment', async assert => {
 
 })
 
+describe('Migrate cbs and rep for orgs', async assert => {
```
Comment (n13) [inline test/accounts.test.js:1747]: Migration unit tests are great! Thanks!! 🙏🏻

**Review by n13 — COMMENTED** [review]
Great stuff

**Comment by n13** [comment]
Oh I opened this PR originally so I can't approve it ;)

**Comment by n13** [comment]
Approved!

**Pull request merged**